docs/assisted-deployment.md

```diff
@@ -8,13 +8,15 @@ the Assisted Service documentation for more information about its architecture a
 Pre-requisites
 ==
 
-2 disks need to be created and added to the worker nodes. This can be done either using
-`virt-manager` or libvirt directly. The disks should be of, at least, 10GB each and they should be
-attached to the `/dev/sdb` and `/dev/sdc` devices respectively. Please update
-`assisted_deployment.sh` if this is not the case.
-
-Note that there is work in progress to automate the disk creations and the device name
-configuration. More info in [this issue](https://github.com/openshift-metal3/dev-scripts/issues/1228)
+2 disks need to be created and added to the worker nodes. The disks should be
+of, at least, 10GB each. Set the following variables in your `config.sh`
+before configuring the development environment:
+
+```bash
+export VM_EXTRADISKS=true
+export VM_EXTRADISKS_LIST="vdb vdc"
```

Comment:
Ref https://github.com/openshift-metal3/dev-scripts/pull/1313/commits/75a147b58ac48c7fb5bb4e5ab1f271c0f8cf3ceb I don't think this list will work - in my testing the extra disk shows up as vda, with the primary OS disk being sda

Reply:
Can confirm :(